File: Pandacap.Data/ProfileProperty.cs
using System.ComponentModel.DataAnnotations;

namespace Pandacap.Data
{
    public class ProfileProperty
    {
        public Guid Id { get; set; }

        [Required]
        public string Name { get; set; } = "";

        [Required]
        public string Value { get; set; } = "";

        public string? Link { get; set; }
    }
}

File: Pandacap.Data/ProfileProperty.fs
namespace Pandacap.Data

open System
open System.ComponentModel.DataAnnotations

type ProfileProperty() =

    member val Id = Guid.Empty with get, set

    [<Required>]
    member val Name = "" with get, set

    [<Required>]
    member val Value = "" with get, set

    member val Link = nullString with get, set
